packages/flutter/test/widgets/overscroll_indicator_test.dart

```diff
@@ -383,13 +430,25 @@ void main() {
         ),
       );
       final RenderObject painter = tester.renderObject(find.byType(CustomPaint));
-      await slowDrag(tester, const Offset(200.0, 200.0), const Offset(0.0, 5.0));
+      await slowDrag(tester, const Offset(200.0, 200.0), const Offset(0.0, 5.0)); // offset will be magnified ten times
       expect(painter, paints..save()..translate(y: 50.0)..scale()..circle());
-      // Reverse scroll direction.
+      // Reverse scroll (30 pixels), and the offset < notification.paintOffset.
       await tester.dragFrom(const Offset(200.0, 200.0), const Offset(0.0, -30.0));
       await tester.pump();
-      // The painter should follow the scroll direction.
+      // The painter should follow the scroll direction .
       expect(painter, paints..save()..translate(y: 50.0 - 30.0)..scale()..circle());
+
+      // Regressing test for https://github.com/flutter/flutter/issues/64149
```

Comment:
Can you separate this into a different test?

Reply:
Done.